lldb/source/Plugins/Platform/Android/PlatformAndroid.cpp

```diff
@@ -26,6 +24,7 @@ using namespace lldb;
 using namespace lldb_private;
 using namespace lldb_private::platform_android;
 using namespace std::chrono;
+using namespace llvm;
```

Comment:
I'd rather not add this. I don't see any heavy llvm usage, and this file has enough using directives already.